portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -31,6 +31,19 @@
   private static final String ENTITY_PROPERTY_KEY_6 = "address";
   private static final String ENTITY_PROPERTY_KEY_7 = "description";
 
+  class Result {
+    private StoreStatus status;
+    private StoreFailureType failure; 
+
+    public void setStatus(StoreStatus status) {
```

Comment:
Instead of creating setter for each field, just have a constructor that gets the value for the two fields and sets it.

Considering these fields are private, not sure, how they are going to be used?